README.md

```diff
@@ -418,24 +418,44 @@ pgbench_command: pgbench -c 32 -j 16 -T <test time in seconds> -P 10 -r
 
 ## <a name="running-automated-hammerdb-benchmark"></a>Running Automated Hammerdb Benchmark
 
-Hammerdb tests are run from a driver node. Driver node is in the same virtual network as the cluster.
-You can customize the hammerdb cluster in the `hammerdb` folder using `azuredeploy.parameters.json`.
-
 **You should create a new branch and change the settings in the new branch and push the branch so that
 when the tool clones the repository it can download your branch.**
 
+Hammerdb tests are run from a driver node. Driver node is in the same virtual network as the cluster.
+You can customize the hammerdb cluster in the `hammerdb` folder using `hammerdb/azuredeploy.parameters.json`.
+Note that this is the configuration for the cluster, which is separate than benchmark configurations(`fabfile/hammerdb_confs/`)
+
+In `fabfile/hammerdb_confs` you can(and you should probably add at least one more config to this folder):
+
+* change postgres version
+* use enterprise or community
+* use a custom branch
+* change/add postgres/citus settings
+
+You can add as many configs as you want to `fabfile/hammerdb_confs` folder and the automation tool will
+run the benchmark for each config. It will clean all the tables in each iteration to get more accurate results.
+So if you want to compare two branches, you can create two identical config files with two different branches.
+The result logs will contain the config file so that it is easy to know which config was used for a run.
+
+After adding the configs `fabfile/hammerdb_confs` could look like:
+
+* ./master.ini
+* ./some_branch.ini
+* ./some_other_branch.ini
+
 In order to run hammerdb benchmark:
 
 ```bash
 eval `ssh-agent -s`
 ssh-add
 export RESOURCE_GROUP_NAME=<your resource group name>
 export GIT_USERNAME=<Your github username>
-export GIT_TOKEN=<Your github token with at least write and read access> # You can create a github token from https://github.com/settings/tokens.
+export GIT_TOKEN=<Your github token with write:packages and read:packages permissions> # You can create a github token from https://github.com/settings/tokens.
 cd hammerdb
-# YOU SHOULD CREATE A NEW BRANCH AND CHANGE THE SETTINGS IN THE NEW BRANCH
+# YOU SHOULD CREATE A NEW BRANCH AND CHANGE THE SETTINGS/CONFIGURATIONS IN THE NEW BRANCH
 # AND PUSH THE BRANCH SO THAT WHEN THE TOOL CLONES THE REPOSITORY
 # IT CAN DOWNLOAD YOUR BRANCH.
+vim fabfile/hammerdb_confs/<branch_name>.ini # verify that your custom config file is correct
 ./create-run.sh
```

Comment:
As suggested by @serprex, it makes sense to put some command like `echo "successfully started the benchmark"` to the end of ./create-run.sh so that silent failures won't go invisible. For example ssh-keyscan could fail without giving any error message.

Reply:
Addressed this, merging